ocs_ci/utility/utils.py

```diff
@@ -366,7 +366,9 @@ def run_cmd(cmd, **kwargs):
         (str) Decoded stdout of command
 
     """
-    log.info(f"Executing command: {cmd}")
+    silence_logs = kwargs.pop('silent', False)
```

Comment:
kwargs.get() instead,  How about *quite* as an option?

Reply:
Using `.get()` creates a problem later on, because `kwargs` are passed over to `subprocess.run()` which doesn't know how to handle the `silent` option and crashes. Pop gives the same safety (if it doesn't exist, it returns False), while also returning `kwargs` to their original state.
Regarding `quiet` - sounds good, I'll implement that.